docs/input-jdbc.asciidoc

```diff
@@ -373,6 +374,30 @@ result-set. The limit size is set with `jdbc_page_size`.
 
 Be aware that ordering is not guaranteed between queries.
 
+[id="plugins-{type}s-{plugin}-jdbc_paging_avoid_count"]
+===== `jdbc_paging_avoid_count`
+
+  * Value type is <<boolean,boolean>>
+  * Default value is `false`
+
+Whether to use count query during the JDBC paging
+
+If false, a count query followed by multiple paged queries (with `LIMIT` statement, etc.) are used under the hood.
+
+If true, multiple paged queries (without a count query ahead) will be performed until no more rows are retrieved.
+The `offset` and `size` parameters must be used in your statement (`size` equal to `jdbc_page_size`, and `offset` incremented by `size`).
+Example:
+
+[source, ruby]
+------------------------------------------------------
+"SELECT id, mycolumn1, mycolumn2 FROM my_table WHERE id > :sql_last_value LIMIT :size OFFSET :offset"
+------------------------------------------------------
+
+[source, ruby]
+------------------------------------------------------
+"CALL fetch_my_data(:sql_last_value, :offset, :size)"
+------------------------------------------------------
+
```

Comment:
Please move the new option/info above `jdbc_paging_enabled` to keep options and sections in alpha order.

Reply:
Thanks for this info and the examples. The examples help.